deploy.rb

```diff
@@ -29,11 +29,15 @@ def on_standby?
 
 dep 'yarn packages installed', :path do
   met? {
-    output = raw_shell('yarn check', :cd => path)
-    output.ok?
+    # We haven't found the right way to have yarn return non-zero when there's packages
+    # to install. As an interim measure, we're falling back to npm to check if
+    # work is required.
+    output = raw_shell('npm ls', :cd => path)
+    # Older `npm` versions exit 0 on failure.
+    output.ok? && output.stdout['UNMET DEPENDENCY'].nil?
```

Comment:
I'm not sure this part of the change was right. [The docs](https://yarnpkg.com/en/docs/cli/check) for `yarn check` are pretty minimal:

> Verifies that versions of the package dependencies in the current project’s package.json matches that of yarn’s lock file.

.. but I've run some checks locally and `yarn check` definitely returns non-zero if packages are missing from `node_modules`.

An open question is whether we need to include the `--production=false` flag to make it check `devDependencies` though.

/cc @nickbrowne